.github/workflows/govcloud.yml

```diff
@@ -109,31 +152,124 @@ jobs:
 
-      - name: ZAP Full Scan
+  security:
+    needs: deploy_dev
+    runs-on: ubuntu-latest
+    environment: Dev
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.10.1
+      - name: install dependencies
+        run: python -m pip install --upgrade pip && python -m pip install -r requirements.txt
+      - name: Bandit
+        run: bandit -r ./app
+      - name: Safety
+        run: safety check
+      - name: ZAP Scan
         uses: zaproxy/action-full-scan@v0.3.0
         with:
-          target: ${{ secrets.FORMS_SERVICE_HOST }}
+          target: "https://funding-service-design-frontend-dev.london.cloudapps.digital"
           allow_issue_writing: False
+
+  deploy_test:
+    needs: security
+    runs-on: ubuntu-latest
+    environment: test
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.10.1
+      - name: create python env
+        run: python -m venv .venv
+      - name: install dependencies
+        run: source .venv/bin/activate && python -m pip install --upgrade pip && pip install -r requirements.txt
+      - name: build static assets
+        run: python build.py
+      - name: Deploy to Gov PaaS
+        uses: citizen-of-planet-earth/cf-cli-action@v2
+        with:
+          cf_api: ${{secrets.CF_API}}
+          cf_org: ${{secrets.CF_ORG}}
+          cf_space: ${{secrets.CF_SPACE }}
+          cf_username: ${{secrets.CF_USER}}
+          cf_password: ${{secrets.CF_PASSWORD}}
+          command: push funding-service-design-frontend-test
+
+  # run_performance_tests:
+  #   needs: deploy_test
+  #   runs-on: ubuntu-latest
+  #   environment: test
+  #   steps:
+  #     - name: checkout code
+  #       uses: actions/checkout@main
+  #       with:
+  #         repository: communitiesuk/funding-service-design-performance-tests
+  #         path: ./funding-service-design-performance-tests
+  #     - name: Set up Python
+  #       uses: actions/setup-python@v2
+  #       with:
+  #         python-version: 3.10.1
+  #     - name: create python env
+  #       run: python -m venv .venv
+  #     - name: install dependencies
+  #       run: source .venv/bin/activate && python -m pip install --upgrade pip && pip install -r ./funding-service-design-performance-tests/requirements.txt
+
+  #     - name: Run performance tests
+  #       run: ls && python -m pip install locust && python -m locust
+  #       working-directory: ./funding-service-design-performance-tests
+  #     - name: "Upload Artifact"
+  #       uses: actions/upload-artifact@v2
+  #       with:
+  #         name: performance-test-report
+  #         path: ./funding-service-design-performance-tests/locust_html_report.html
+  #         retention-days: 5
```

Comment:
As above, was it intentional to comment out this block? can it be either uncommented or removed?

Reply:
The performance tests are awaiting @rayhan15243 changes—so on pause for now.
The runner testing fails on the 404 error, I believe the fix is underway on this repo.

Reply:
OK, can we add a TODO then to uncomment this when the tests are fixed?